templates/webapps/galaxy/workflow/run.mako

```diff
@@ -583,9 +583,10 @@ if wf_parms:
                     <%
                     pja_ss_all = []
                     for pja_ss in [ActionBox.get_short_str(pja) for pja in step.post_job_actions]:
-                        pja_ss = h.escape( pja_ss )
                         for rematch in re.findall('\$\{.+?\}', pja_ss):
```

Comment:
I may be missing something but there seem to be a lot of seemingly unescaped values that go into ``get_short_str`` right?

```
            return "Add tag(s) '%s' to '%s'." % (pja.action_arguments['tags'], pja.output_name)
```

```
        return "Set the datatype of output '%s' to '%s'" % (pja.output_name, pja.action_arguments['newtype'])
```

```
            return "Rename output '%s' to '%s'." % (pja.output_name, pja.action_arguments['newname'])
```


```
        return "Hide output '%s'." % pja.output_name
```

Or are these things escaped going into the pja?